fabric-events-interaction-v0/src/main/java/net/fabricmc/fabric/mixin/eventsinteraction/MixinServerPlayerEntity.java

```diff
@@ -36,4 +40,13 @@ public void onPlayerInteractEntity(Entity target, CallbackInfo info) {
 			info.cancel();
 		}
 	}
+
+	@Inject(method = "dropItem", at	= @At("HEAD"), cancellable = true)
+	public void onPlayerDropItem(final ItemStack stack, final boolean boolean_1, final boolean boolean_2, final CallbackInfoReturnable<ItemEntity> info) {
```

Comment:
Would be nice to name the parameters.

Reply:
My guess for the second boolean was `hasOwner`. First one seems to be related to the item dropping randomly (maybe `isThrown`)? Not sure. Any thoughts on that?